pycrest/eve.py

```diff
@@ -159,7 +159,7 @@ def __init__(
         if user_agent is None:
             user_agent = "PyCrest/{0} +https://github.com/pycrest/PyCrest"\
                 .format(version)
-        if transport_adapter is not None:
+        if transport_adapter is not None and isinstance(transport_adapter, HTTPAdapter):
```

Comment:
Ahah, right, I don't even know why I left that `if transport_adapter is not None` when I added the check on `isinstance` as checking isinstance is enough...